### PR TITLE
Make VOR available for opt-ins

### PR DESCRIPTION
**Pull request by HazalCiplak**
https://github.com/elifesciences/data-hub-issues/issues/922

### CODE REVIEW

File: data_hub_api/docmaps/v2/codecs/docmaps.py [hunk 1]
```diff
@@ -284,9 +284,8 @@ def iter_docmap_steps_for_query_result_item(query_result_item: ApiInput) -> Iter
                         query_result_item,
                         manuscript_version
                     )
-        else:
-            previous_manuscript_version = query_result_item['manuscript_versions'][index - 1]
-            assert manuscript_version['position_in_overall_stage'] > 1
+        if manuscript_version['vor_publication_date'] and index == len(manuscript_versions) - 1:
+            previous_manuscript_version = manuscript_versions[index - 1]
```
Comment (de-code) [inline data_hub_api/docmaps/v2/codecs/docmaps.py]: Wouldn't this fail if index wasn't at least `1`?

File: tests/unit_tests/docmaps/v2/codecs/docmaps_test.py [hunk 2]
```diff
@@ -563,25 +577,25 @@ def test_should_populate_assertions_for_second_manuscript_published_step(self):
             'status': 'manuscript-published'
         }]
 
-    def test_should_populate_inputs_for_vor_published_step(self):
+    def test_should_populate_inputs_for_vor_published_step_for_opt_ins(self):
+        manuscript_versions = [
+            {**MANUSCRIPT_VERSION_WITH_EVALUATIONS_1,
```
Comment (de-code) [inline tests/unit_tests/docmaps/v2/codecs/docmaps_test.py]: minor: might be easier to read if it was all on one line or curly brackets on separate lines?

i.e. one of the following

```python
manuscript_versions = [{
    **MANUSCRIPT_VERSION_WITH_EVALUATIONS_1,
    'vor_publication_date': VOR_PUBLICATION_DATE_1
}]
```

Or

```python
manuscript_versions = [
    {
        **MANUSCRIPT_VERSION_WITH_EVALUATIONS_1,
        'vor_publication_date': VOR_PUBLICATION_DATE_1
    }
]
```

Or

```python
manuscript_versions = [
    {**MANUSCRIPT_VERSION_WITH_EVALUATIONS_1, 'vor_publication_date': VOR_PUBLICATION_DATE_1}
]
```

What do you think?

File: tests/unit_tests/docmaps/v2/test_data.py [hunk 3]
```diff
@@ -15,6 +15,7 @@
 LONG_MANUSCRIPT_ID_2 = 'long_manuscript_id_R1'
 
 LONG_MANUSCRIPT_VOR_ID_1 = 'long_manuscript-VOR-id'
+LONG_MANUSCRIPT_VOR_ID_FOR_OPT_INS_1 = 'long_manuscript-id-R1'
```
Comment (de-code) [inline tests/unit_tests/docmaps/v2/test_data.py]: Do we still need this?

**Comment by HazalCiplak** [comment]
Hi @fred-atherden,
We have created example DocMaps for Opt-Ins with `vor-published` step. Could you please check the manuscript versions in the vor-published step for the files for  '84553', '85596', '80984' ?

**Comment by fred-atherden** [comment]
Thanks @HazalCiplak. They look good to me!

**Pull request merged**